solutions/USACO jan19-mountain-view.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: usaco-896
+source: USACO Silver 2019 January
+title: Mountain View
+author: Jesse Choe
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_mountains_silver_jan19.html)
+
+**Time Complexity:** $\mathcal{O}(n\log n)$
+
+<LanguageSection>
+<CPPSection>
+
+## Sorting Struct Using Comparator Function
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define pb push_back
+#define all(x) begin(x), end(x)
+#define f first
+#define s second
+
+void setIO(string fileName = ""){
+	FastIO ios_base::sync_with_stdio(0); cin.tie(0);
```

Comment:
why is there FastIO here?

Reply:
Idk I just use it in my template lol. I copied and pasted this from somewhere, and I've been using this since. :confused:

Reply:
does this compile??

Reply:
Yeah, I think so. I haven't tried running my detemplified code yet. Lemme try rn.

Reply:
no... the `FastIO` macro was removed so you should remove it

Reply:
ohhh I see what u mean. i forgot to remove this when detemplifying.

Reply:
Ok it works now.